src/main/java/mil/dds/anet/beans/Person.java

```diff
@@ -224,10 +227,30 @@ public Position getPosition() {
         });
   }
 
+  public CompletableFuture<List<PersonPositionHistory>> loadAllPeoplePositionHistory() {
+    if (allPeoplePositionHistory != null) {
+      return CompletableFuture.completedFuture(allPeoplePositionHistory);
+    }
+    Map<String, Object> context = AnetObjectEngine.getInstance().getContext();
+    return AnetObjectEngine.getInstance().getPersonDao().getAllPositionHistory(context, uuid)
+        .thenApply(o -> {
+          allPeoplePositionHistory = o;
+          return o;
+        });
+  }
+
   public List<PersonPositionHistory> getPreviousPositions() {
     return previousPositions;
   }
 
+  @JsonIgnore
+  public List<PersonPositionHistory> getAllPeoplePositionHistory() {
```

Comment:
This method is only called by the tests, so really belongs somewhere under `src/test/java`.

Reply:
This method is moved under `src/test/java`.